src/Model/Transform.py

```diff
@@ -0,0 +1,44 @@
+import numpy
+
+from src.constants import DEFAULT_WINDOW_SIZE
+
+
+# A safe way of dividing numbers
+def safe_division_transform(x, y, n_x, n_y, d_x, d_y):
```

Comment:
Docstrings missing in this file.

Reply:
Just saw this as I committed docstrings haha, thanks for picking up on it!